src/Razor/Microsoft.AspNetCore.Razor.Tools/src/BrotliCompressCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Threading.Tasks;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace Microsoft.AspNetCore.Razor.Tools
+{
+    internal class BrotliCompressCommand : CommandBase
+    {
+        public BrotliCompressCommand(Application parent)
+            : base(parent, "brotli")
+        {
+            Sources = Option("-s", ".cshtml files to compile", CommandOptionType.MultipleValue);
+            Outputs = Option("-o", "Generated output file path", CommandOptionType.MultipleValue);
+            CompressionLevelOption = Option("-c", "Compression level", CommandOptionType.SingleValue);
```

Comment:
I don't really feel strongly about removing this, but I'm not aware we have any use case for having an option. We always want to use max AFAIK.

Reply:
They get turned down in the Razor integration tests. It makes the tests take far too long if they perform maximal compression every time.